source/interprocedural_analyses/taint/modelParser.ml

```diff
@@ -23,6 +23,61 @@ open Statement
 open Domains
 open ModelParseResult
 
+module PythonVersion = struct
```

Comment:
That's a good suggestion :)

Reply:
@arthaud are you sure this works? ModelParser.PythonVersion.major then is unbound.
NB: I also modified the mli file so that `t = Configuration.PythonVersion.t`